layouts/partials/author/articles_list.html

```diff
@@ -20,7 +20,7 @@ <h2 class="article-title">
                     {{ .Params.excerpt }}
                   </p>
                   <div class="article-metadata">
-                    {{ .Date.Format "January 2, 2006" }}{{ if .Params.timetoread }} • {{ .Params.timetoread }} min read{{ end }}
+                    {{ .Date.Format "January 2, 2006" }} hey • {{ .ReadingTime }} min read
```

Comment:
hey?

Reply:
A lost bit of testing, sorry about that.